Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
```

Comment:
Consider adding error handling for commands since failure of `go run`, `go test`, or `goi18n` will not produce clear feedback if an error occurs. Wrapping commands in a shell script that checks the exit status, or using a task runner that supports error checks can significantly improve debugging and usability.